app/uk/gov/hmrc/cgtpropertydisposals/connectors/dms/GFormConnector.scala

```diff
@@ -74,10 +75,18 @@ class GFormConnectorImpl @Inject() (playHttpClient: PlayHttpClient, servicesConf
               case status if is4xx(status) || is5xx(status) =>
                 logger.warn(s"Bad response status from gform service ${response.body}")
                 Left(Error(response.body))
-              case _                                        => Left(Error("Invalid HTTP response status from gform service"))
+              case _                                        =>
+                logger.warn("could not send dms payload to GFORM service")
+                Left(Error("Invalid HTTP response status from gform service"))
             }
           }
+          .recover {
+            case e: Exception =>
```

Comment:
it'd be safer to match against `NonFatal(e)` here

Reply:
fixed